cv32/sim/xrun/indago.tcl

```diff
@@ -0,0 +1,6 @@
+# Indago waveform probe script
```

Comment:
This is exactly the kind of file I was expecting to see in this directory.   I have thought for some time that these should be in `cv32/sim/tools/xrun` (not `cv32/sim/xrun`) since this would keep the _sim_ directory a little cleaner.   What do you think?

Reply:
Agree with this.  I will amend pull request to move those directories.  Other folks have a few scripts there so we will need to communicate the change.